<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_update_teacher_schedule</class>
 <widget class="QWidget" name="admin_update_teacher_schedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1659</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">color: #111;
    font-size: 16px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;;
    background-color: transparent;
    padding-bottom: 5px;
   </string>
       </property>
       <property name="text">
        <string>Search by Teacher_Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchbyteacherid">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 2px solid #a0eec0;
    border-radius: 20px;
    padding: 8px 15px;
    font-size: 14px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #ff8591;
    background-color: #f9f9f9;
}

QLineEdit::placeholder {
    color: #95a5a6;
    font-style: italic;
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter Teacher ID</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">color: #111;
    font-size: 16px;
    font-weight: bold;
    font-family: &quot;Segoe UI&quot;;
    background-color: transparent;
    padding-bottom: 5px;
   </string>
       </property>
       <property name="text">
        <string>Search by Teacher Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchbyteacheremail">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 2px solid #a0eec0;
    border-radius: 20px;
    padding: 8px 15px;
    font-size: 14px;
    color: #2c3e50;
}

QLineEdit:focus {
    border: 2px solid #ff8591;
    background-color: #f9f9f9;
}

QLineEdit::placeholder {
    color: #95a5a6;
    font-style: italic;
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter Teacher Email</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableWidget" name="showteacherstable">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QTableWidget {
    background-color: #ffffff;
    alternate-background-color: #f2f2f2;
    selection-background-color: #a0eec0;
    selection-color: #000000;
    border: 1px solid #dcdcdc;
    gridline-color: #dcdcdc;
}

QHeaderView::section {
    background-color: #f8f9fa;
    padding: 4px;
    border: 1px solid #dcdcdc;
    font-weight: bold;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(203, 255, 246);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
font: 11pt &quot;Segoe UI&quot;;</string>
           </property>
           <property name="text">
            <string>Update  assigned classes</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="verticalWidget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>400</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border color : black;
background-color: rgb(199, 233, 255);
</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="styleSheet">
               <string notr="true">QRadioButton {
    color: #333333;
    font-size: 14px;
    spacing: 10px;
}

QRadioButton:checked {
    color: #0078d7;
    font-weight: bold;
}

QRadioButton::indicator {
    width: 18px;
    height: 18px;
    border-radius: 11px;
    border: 2px solid #999999;
    background-color: #ffffff;
}

QRadioButton::indicator:hover {
    border: 2px solid #0078d7;
}

QRadioButton::indicator:checked {
    background-color: #0078d7;
    border: 2px solid #005a9e;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Teacher Email</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit"/>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Teacher Id</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_2"/>
               </item>
               <item>
                <widget class="QLabel" name="label_24">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>12</pointsize>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Department</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QRadioButton" name="radioScience">
                    <property name="text">
                     <string>Science</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioCommerce">
                    <property name="text">
                     <string>Commerce</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_25">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>12</pointsize>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Subjects</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="widget_4" native="true">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QGroupBox" name="Sciencesubjects">
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(171, 255, 209);</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <widget class="QRadioButton" name="radioMath">
                       <property name="text">
                        <string>Math</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioBiology">
                       <property name="text">
                        <string>Biology</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioChemistry">
                       <property name="text">
                        <string>Chemistry</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioPhysics">
                       <property name="text">
                        <string>Physics</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QGroupBox" name="Commercesubjects">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background-color: rgb(197, 222, 255);</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QRadioButton" name="radioEconomics">
                       <property name="text">
                        <string>Economics</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioBusiness">
                       <property name="text">
                        <string>Business</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioFinance">
                       <property name="text">
                        <string>Finance</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="radioAccountancy">
                       <property name="text">
                        <string>Accountancy</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_26">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>12</pointsize>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Class</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QRadioButton" name="radio11">
                    <property name="text">
                     <string>11</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radio12">
                    <property name="text">
                     <string>12</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_27">
                 <property name="font">
                  <font>
                   <family>Segoe UI</family>
                   <pointsize>12</pointsize>
                   <italic>false</italic>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;Segoe UI&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Sections</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="styleSheet">
                  <string notr="true">color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QRadioButton" name="radioA">
                    <property name="text">
                     <string>A</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="radioB">
                    <property name="text">
                     <string>B</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="tdprofile_addtoschedule">
                 <property name="styleSheet">
                  <string notr="true">background-color: rgb(255, 115, 152);
color: rgb(0, 0, 0);
font: 600 12pt &quot;Segoe UI&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Add to your schedule</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="tempScheduleTable">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(130, 180, 255);
background-color: rgb(230, 228, 255);
border-radius: 25%
</string>
              </property>
              <column>
               <property name="text">
                <string>Subject</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Department</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Class</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Section</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="savebutton2">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>50</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
font: 600 9pt &quot;Segoe UI&quot;;
background-color: rgb(255, 133, 149);</string>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
